### PR TITLE
Sign Up Screen

### PR DESCRIPTION
**Pull request by WaterGhost1**
View - SignUpScreen - init, py, kv
Controller - init, signup_screen.py
Model - init, signup_screen.py
screens.py

### CODE REVIEW

File: View/screens.py [hunk 1]
```diff
@@ -3,24 +3,27 @@
 of the screens of the application.
 """
 
-from Controller import (LoginScreenController, MainScreenController,
```
Comment (sbstntmprl) [inline View/screens.py:6]: Missing _SignUpController._

File: View/screens.py [hunk 2]
```diff
@@ -3,24 +3,27 @@
...
-                        ProfileScreenController, SignupScreenController)
-from Model import LoginScreenModel, MainScreenModel, ProfileScreenModel, SignupScreenModel
```
Comment (sbstntmprl) [inline View/screens.py]: Missing _SignupModel_

File: Controller/signup_screen.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        self.views[0].ids.user.text = ""
+        self.views[0].ids.passw.text = ""
+        self.views[0].ids.conpass.text = ""
+
+    def disabled_btn(self):
+        if self.views[0].ids.btn.disabled == False:
+            self.views[0].ids.btn.disabled =True
+    
+    def pass_data(self):
+        self.userdata = self.views[0].get_userdata()
```
Comment (Lemon1903) [inline Controller/signup_screen.py:39]: yung `self.userdata` ba is diyan lang yung scope? if yes, gawin mo nalang userdata

File: Controller/signup_screen.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        self.model.to_database(self.userdata)
+
+    def check_clr_disable(self):
```
Comment (Lemon1903) [inline Controller/signup_screen.py:42]: pakibuo ng clr tenkyu, kung ano man yon

File: Controller/signup_screen.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        self.pass_data()
+        self.disabled_btn()
+        self.clear_txt_field()
+
+    def check_txt_field(self):  #bind sa kv file
+        self.usern = self.username_check()
```
Comment (Lemon1903) [inline Controller/signup_screen.py:48]: what is usern? pakibuo tenkyu

File: Controller/signup_screen.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+        elif self.views[0].ids.user.text == self.usern:
+            print("username taken")
+            self.views[0].error_user_taken()
+
+        elif self.views[0].ids.passw.text == "":
+            print("Fill Password")
+            self.views[0].error_pass()
+
+        elif self.views[0].ids.conpass.text == "":
```
Comment (Lemon1903) [inline Controller/signup_screen.py:62]: conpass pakibuo ulit tenkyu

File: Controller/signup_screen.py [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+            self.views[0].error_conpass()
+
+        elif self.views[0].ids.passw.text != self.views[0].ids.conpass.text:
+            print("Password Do not Match")
+            self.views[0].error_notmatch()
+
+        else:
+            self.check_clr_disable()
+    
+    def username_check(self):
```
Comment (Lemon1903) [inline Controller/signup_screen.py:73]: boolean return value ba to?

Comment (Lemon1903) [inline Controller/signup_screen.py:73]: boolean return value ba to?

**Pull request merged**